### PR TITLE
[TEST] Add more test cases to mini example: 

### PR DESCRIPTION
**Pull request by Irallia**
Resolves a part of #144.
More test cases in mini example: Translocation and Duplications
plus the new IGV view:
single_end_IGV_View

### CODE REVIEW

File: test/data/mini_example/explanation.fasta [hunk 1]
```diff
@@ -1,23 +1,28 @@
...
+CGCCCATGCAACTAGCGATGCTAGCTAGCTAGCTTACGACTGGCCATGCG                                          breakpoints:
+GCGGCAT ATACATAAGGGGCTCATCGATCGATTTCGG                              <- Deletion             ref: (57, 70] deleted
+ATCGGGGGGCCCCCATTTTAAACGG CCCCGGGGCCAATTT GGATCTTGACTCTGGAAAACTTTTA <- Insertion            ref: after 125
+ACGCCGGGAATCGGTAGTCCTTTCGCGGGG ATATATTT ATATATTT TACGGGATATAA       <- Tandem Duplication   ref: [180, 188] duplicated
 CGATCGGATCGGATCGATAATAGGTCTCGGTTGCCAACTGATCGTACCAA
-ATATTTCTGCGGGGCT TCGGCTAACGGTT AGAG CGCCCCTCCGCGATTA C              <- Translocation    ref: (266, 267, 232, 233, 236, 237, 249)
-                                                                                oder    ref: [267, 286] deleted
+ATATTTCTGCGGGGCT TCGGCTAACGGTT AGAG CGCCCCTCCGCGATTA C              <- Double Translocation ref: (266, 267, 232, 233, 236, 237, 249)
+                                                                                oder        ref: [267, 286] deleted
 ACCCATGCGGATTATAAACGGG ATATTAAGGGCTTT                               <- Duplication in der Referenz
```
Comment (eaasna) [inline test/data/mini_example/explanation.fasta:21]: Here's some Denglish in case you want to avoid that :wink:

**Review by eaasna — APPROVED** [review]
Looks good. It's a nice idea to have the explanation.fasta file

As I understand the output_err.txt is what's printed to standard output and output_res.txt is the output file?

File: test/data/mini_example/explanation.fasta [hunk 2]
```diff
@@ -1,23 +1,28 @@
...
+CGTACTGAAAAAAACCAAACGGGGAAATATTAACCGGCCTTTGCCTGGGA
+CGCTGTGACTGTACGGGGGGGCGGGCCTTATT
 
 >new_Creature
-CGCCCATGCAACTAGCGATGCTAGCTAGCTAGCTTACGACTGGCCATGCG                                      breakpoints:
-GCGGCAT ATACATAAGGGGCTCATCGATCGATTTCGG                              <- Deletion         ref: [57, 70] deleted
-ATCGGGGGGCCCCCATTTTAAACGG CCCCGGGGCCAATTT GGATCTTGACTCTGGAAAACTTTTA <- Insertion        ref: after 125 something inserted
-ACGCCGGGAATCGGTAGTCCTTTCGCGGGG ATATATTT ATATATTT TACGGGATATAA       <- Duplication      ref: (180 or 189) something inserted
+CGCCCATGCAACTAGCGATGCTAGCTAGCTAGCTTACGACTGGCCATGCG                                          breakpoints:
+GCGGCAT ATACATAAGGGGCTCATCGATCGATTTCGG                              <- Deletion             ref: (57, 70] deleted
```
Comment (joergi-w) [inline test/data/mini_example/explanation.fasta:15]: Attention: equivalent to the previous would be `(56` here, so I guess it was mistaken before?

File: test/data/mini_example/explanation.fasta [hunk 3]
```diff
@@ -1,23 +1,28 @@
...
+ATCGGGGGGCCCCCATTTTAAACGG CCCCGGGGCCAATTT GGATCTTGACTCTGGAAAACTTTTA <- Insertion            ref: after 125
+ACGCCGGGAATCGGTAGTCCTTTCGCGGGG ATATATTT ATATATTT TACGGGATATAA       <- Tandem Duplication   ref: [180, 188] duplicated
```
Comment (joergi-w) [inline test/data/mini_example/explanation.fasta:17]: the same here, now 180 is included, before it wasn't

**Review by joergi-w — APPROVED** [review]
Looks good! I marked two places where there could be a possible mistake with the boundaries. Please double-check :)

**Pull request merged**